### PR TITLE
Updates fibers to 3.0 to support node >=10

### PR DESCRIPTION
**Pull request by BorisOsipov**
Updates fibers to 3.0 to support node >=10

Fixed https://github.com/webdriverio/wdio-cucumber-framework/issues/153

### CODE REVIEW

**Comment by christian-bromann** [comment]
Something with the Babel compiling went wrong:
```
/home/travis/build/webdriverio-boneyard/wdio-sync/test/addCommand.spec.js:3
import _Object$keys from 'babel-runtime/core-js/object/keys';
^^^^^^
SyntaxError: Unexpected token import
```

**Comment by BorisOsipov** [comment]
Yep, it doesn't work at master for me. Will fix.

File: .babelrc [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
-    }],
-    "stage-0"
-  ],
-  "plugins": [
-    "rewire",
-    "transform-runtime",
-    "add-module-exports",
+    {
+      "plugins": [
+        "rewire",
```
Comment (christian-bromann) [inline .babelrc]: let's only use this plugin in testing environment

File: .babelrc [hunk 2]
```diff
@@ -1,15 +1,24 @@
 {
+  "passPerPreset": true,
```
Comment (christian-bromann) [inline .babelrc]: not sure if we need this option, happy to bump up the node target to 8

**Comment by BorisOsipov** [comment]
Сan not fix it. Current package.json with updated fibers produce a real mess of babel 6 and 7.beta dependencies. Don't have time to deal with it.

**Pull request closed without merging**